quiz.js

```diff
@@ -25,7 +38,33 @@ if (square(2) !== 4 || cube(3) !== 27 || quad(4) !== 256) {
 // If the flag is falsy but not undefined, it should return half that sum
 
 /* your code starts here */
+var nums = [2, 4, 6];
+
+var previousValue = 0;
+var currentValue = 0;
+var total = 0;
 
+function sum(nums, twiceOrHalf) {
+  for (var i = 0; i < nums.length; i++) {
```

Comment:
I think you misunderstood the question. You should be totalling all of the elements in nums THEN applying the twiceOrHalf changes